_config.yml

```diff
@@ -17,8 +17,9 @@ title: Raspberry Pi Documentation
 description: >- # this means to ignore newlines until "baseurl:"
   Raspberry Pi Documentation.
 baseurl: "/documentation" # the subpath of your site, e.g. /blog
-url: "" # the base hostname & protocol for your site, e.g. http://example.com
```

Comment:
I'm sure that there must have been some reason why this was left empty; but that reason is probably now lost to the mists of time...

Reply:
I _think_ it might have an impact on the URLs used for the rptl header. Need to double-check the behaviour on a staging build before I release!